pjrt/convert/utils/trace_utils.zig

```diff
@@ -0,0 +1,14 @@
+// Constants used as trace_viewer PID (device_id in trace_events.proto).
+// PID 0 is unused.
+// Support up to 500 accelerator devices.
+pub const kFirstDeviceId = 1;
```

Comment:
just inline this inside "trace_container.zig" also no need for the "k" prefix. and use snake_case